### PR TITLE
Added some additional websites and community pools to monitor

### PR DESCRIPTION
**Pull request by Primate411**

### CODE REVIEW

**Comment by fallengravity** [comment]
Hey, It appears you've disabled edits from maintainers (Maybe?) - I think so since I'm unable to edit the files because of this you will need to view my comments regarding some adjustments that need to be made before we can merge this PR.

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official & Secondary to Basic

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official & Primary to blockscout

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor:63]: Please change Main to Official

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please remove this pool. We do not plan on integrating other pools since there are constantly new pools and pools that shutdown it is too much to maintain especially since this bot alerts us when one of the services is offline. It'd become quite overwhelming to deal with pools and unfair to the other pool ops.

File: server-monitor
Comment (fallengravity) [inline server-monitor]: Please remove this pool. We do not plan on integrating other pools since there are constantly new pools and pools that shutdown it is too much to maintain especially since this bot alerts us when one of the services is offline. It'd become quite overwhelming to deal with pools and unfair to the other pool ops.

**Review by fallengravity — CHANGES_REQUESTED** [review]
Once these changes have been implemented, I'll review again for any other changes needed & approve if good.

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official and Secondary to Basic

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official and Primary to Blockscout

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor:63]: Changed Main to Official

File: server-monitor
Comment (Primate411) [reply server-monitor]: Removed pool

File: server-monitor
Comment (Primate411) [reply server-monitor]: Removed pool.  

The though process was other pool ops can choose to open a pull request like Dylie instructed.

**Review by fallengravity — APPROVED** [review]

**Pull request merged**
